bindata/network/openshift-sdn/sdn.yaml

```diff
@@ -70,7 +70,7 @@ spec:
           set -euo pipefail
 {{- if .IsNetworkTypeLiveMigration }}
           set -x
-          NODE_CNI="OpenShiftSDN"
+          export NODE_CNI="OpenShiftSDN"
```

Comment:
this is a bit confusing:
```     
         export NODE_CNI="OpenShiftSDN"
         if ip link show br-ex; then
            echo "br-ex exists"
            NODE_CNI="OVNKubernetes"
         else
            echo "br-ex doesn't exist"
         fi
```
can we keep `NODE_CNI` exported in both cases?

Reply:
Done